docs/DeveloperGuide.md

```diff
@@ -323,6 +323,124 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case resumes at step 1.
 
+**Use case: UC2 - Edit contact details**
+
+**MSS:**
+<p>
+1. User requests to add a specific field for the contact. <br>
+2. Reache saves the specific field. <br>
+&emsp;Repeat steps 1 and 2 until satisfied. <br>
+3. User requests to leave ‘edit’ mode. <br>
+4. Reache returns to ‘default’ mode. <br>
+Use case ends.
+
+**Extensions:** <br>
+<p>
+1a. User inputs the wrong format. <br>
+&emsp; 1a1. Reache displays an error message. <br>
+&emsp; Use case resumes from step 1.
+</p>
+
+**Use case: UC3 - Delete a Contact**
+
+**MSS:**
+<p>
+1. User requests to delete a contact. <br>
+2. Reache asks for confirmation. <br>
+3. User confirms deletion. <br>
+4. Reache deletes the contact. <br> 
+&emsp;Repeat steps 1 and 2 until satisfied. <br>
```

Comment:
Is the 'repeat steps' line necessary for use cases except UC2?

Reply:
You're right - it seems redundant. I'll remove them.